.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.7", "3.8"]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: pip-${{ hashFiles('setup.cfg') }}-${{ hashFiles('tox.ini') }}
+          restore-keys: pip-
+
+      - name: Install Tox and any other packages
+        run: pip install tox tox-gh-actions
```

Comment:
This [`tox-gh-actions`](https://github.com/ymyzk/tox-gh-actions) package is designed to serve the same function as [`tox-travis`](https://github.com/tox-dev/tox-travis). It helps smooth over mapping matrix values to Tox environments. This saves us from having to move a lot of the Tox environment logic (like Django variants, lint environment, etc) from the `envlist` to the workflow file.

It looks like there are plans for an official [Tox Action](https://github.com/tox-dev/action) but development doesn't seem to have started yet.

Reply:
Nice find.

Regarding the `pip install` step, it looks like `tox-gh-actions` installs `tox`, so we may be able to omit that from the explicit list of packages installed.

Reply:
Good idea! I'll try that out.